### PR TITLE
Add Interaction sample calculation notebook

### LINKED ISSUE #1391: Establish notebook for interaction plots sample sizes
From [this discussion](https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/262#discussion_r867084046), we need to add a small notebook to the `interaction-plots` module directly reporting relevant sample sizes we discuss in the manuscript.

### PR DESCRIPTION
**Pull request by jharenza**
#### What scientific question is your analysis addressing?
The notebook added here records:
- Total N of PBTA samples included in co-occurrence analysis
- Number of significant events (co-occurrence/mutual exclusivity)
- Number of HGG-specific significant events
- Ratio of hgg/total significant events

#### What was your approach?
Calculate the above from `cooccur_top50.ALL.tsv` and `cooccur_top50.Diffuse-astrocytic-and-oligodendroglial-tumor.tsv`

#### What GitHub issue does your pull request address?
This PR closes #1391 

#### Which areas should receive a particularly close look?
NA

#### Is there anything that you want to discuss further?
No

#### Is the analysis in a mature enough form that the resulting figure(s) and/or table(s) are ready for review?
yes

#### What types of results are included (e.g., table, figure)?
notebook with results

#### What is your summary of the results?
These numbers match those in the manuscript

#### Reproducibility Checklist

- [x] The dependencies required to run the code in this pull request have been added to the project Dockerfile.
- [x] This analysis has been added to continuous integration.

#### Documentation Checklist

- [x] This analysis module has a `README` and it is up to date.
- [x] This analysis is recorded in the table in `analyses/README.md` and the entry is up to date.
- [x] The analytical code is documented and contains comments.

### CODE REVIEW

**Review by sjspielman — COMMENTED** [review]
This looks good!! I'd make one further change here and add a script `run-interaction-plots.R` that can be used to run both the `01-create-interaction-plots.sh` and this notebook, and then add quick info about running the _full_ module to the README.

File: analyses/interaction-plots/README.md [hunk 1]
```diff
@@ -23,3 +23,9 @@ To run a basic analysis and create a co-occurence plot:
 ```
 bash analyses/interaction-plots/01-create-interaction-plots.sh
 ```
+
+# Result Ns used in manuscript
+02-result-ns-for-manuscript.Rmd is a notebook which calculates the relevant Ns and ratios used in the manuscript.
```
Comment (sjspielman) [inline analyses/interaction-plots/README.md:28]:
```suggestion
`02-result-ns-for-manuscript.Rmd` is a notebook which calculates the relevant Ns and ratios used in the manuscript.
```

**Comment by jharenza** [comment]
Yeah good point, will do. I was a little unsure how to add this since the dir structure is a bit different

**Comment by jharenza** [comment]
Yes, didn't see that one, thanks!

**Comment by jharenza** [comment]
@sjspielman ready for re-review. This was a very early module and so the readme doesn't explicity list every script and file as do others, so if we want to update, maybe @jashapiro can update that part. I updated more generally. In addition, I used a bash run script to be consistent with every other module.

**Review by sjspielman — APPROVED** [review]
🚀

**Pull request merged**